notebooks/MAST/PanSTARRS/PS1_DR2_TAP/PS1_DR2_TAP.ipynb

```diff
@@ -491,8 +500,8 @@
     "\"\"\".format(ra,dec,radius)\n",
```

Comment:
Comment for line 481/490:
`Mast._resolve_object(objname)` -> `Mast.resolve_object(objname)`